### ISSUE TITLE
Add navigation properties if needed.

### ISSUE THREAD

**Issue by LunchboxAvenue** [otherw1se/Indepp#38]

**Comment by LunchboxAvenue** [comment]
Fixed in 30c1527df8304d33e6107b6e12d0c7d8967e694f

However, needs investigation on a matter of one-to-one relationship. 
That being said, this resources (http://www.entityframeworktutorial.net/code-first/configure-one-to-one-relationship-in-code-first.aspx) indicates of use of  [Key, ForeignKey("Name")], which obviously won't allow creating an article with null in PlaceId. This is correct, as we want to map an article to a place, however we might also want to map the article to a place later.

**Comment by LunchboxAvenue** [comment]
"There is one type of relationship that Code First will always require configuration for: one-to-one relationships. When you define a one-to-one relationship in your model, you use a reference navigation property in each class. If you have a reference and a collection, Code First can infer that the class with the reference is the dependent and should have the foreign key. If you have two collections, Code First knows it’s many-to-many and the foreign keys go in a separate join table. However, when Code First just sees two references, it can’t work out which class should have the foreign key." (source: https://www.safaribooksonline.com/library/view/programming-entity-framework/9781449317867/ch04s07.html )

In order to keep the functionality that is intended, we are going to use one-to-many relationship, which might even prove to be more useful in the future, as we may want to map several articles to the same place.

But however, if we want to do the one-to-one/zero entity relationship, all we have to do:
1) Create a navigation property (For example in Place Model)

```
public virtual Address Address { get; set; }
```

2) Address model however doesn't need to have it's own Id attribute, but rather have a key and foreign key that comes from Place Model. Also it needs a navigation property to Place.

```
[Key]
[ForeignKey("Place")]
public int PlaceID { get; set; }
public virtual Place Place { get; set; }
```
